<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblcurrentDateTime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>Saturday, 01-01-2022 0:00 AM</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="swLoginPages">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="LoginPage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QPushButton" name="btnLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>205</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);
font: 16px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLoginPassword">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtLoginUsername">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblLoginUsername">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLoginPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>125</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLogin">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 28px</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblNoAccount">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>165</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 12px</string>
      </property>
      <property name="text">
       <string>Don't have an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoginSignup">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);
font: 10px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblLoginIncorrect">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(163, 31, 31); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SignupPage">
     <widget class="QLineEdit" name="txtSignupPassword">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupUsername">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupUsername">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignup">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>490</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);
font: 16px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>125</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignup">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 28px</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupConfirmPassword">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupConfirmPassword">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>175</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Confirm Pass</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupEmail">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>225</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupEmail">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupFirstName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>275</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupFirstName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupLastName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>325</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupLastName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupAge">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>375</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupAge">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>380</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupMobile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>425</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Mobile</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupMobile">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>430</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupCountry">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>275</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupCountry">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>280</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupBusinessCity">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupBusinessCity">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>225</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Business City</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupHomeCity">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>125</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Home City</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupHomeAddress">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>75</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Home Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupHomeAddress">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupBusinessAddress">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>175</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Business Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupHomeCity">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSignupBusinessAddress">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupType">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbCargoOwner">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Cargo Owner</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbDriver">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>400</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Driver</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbTransportationCompany">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>440</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Transportation Company</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignupBack">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>490</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);
font: 16px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignupIncorrect">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>500</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(163, 31, 31); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DriversSignup">
     <widget class="QPushButton" name="btnDriversSignup">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);
font: 16px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDriversSignup">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 28px</string>
      </property>
      <property name="text">
       <string>Drivers Details</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDriversSignupNI">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>NI Number</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDriversSignupNI">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDriversSignupDL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>125</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Driving Licence ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDriversSignupDL">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDriversSignupLorry">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>175</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string>Lorry Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDriversSignupLorry">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(186, 189, 182);
font: 20px</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDriversSignupBack">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>230</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);
font: 16px;
color: rgb(186, 189, 182);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblDriversSignupIncorrect">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(163, 31, 31); 
font-size: 20px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
